app/lib/project/MetapolatorProject.js

```diff
@@ -614,6 +616,38 @@ define([
         exportController.export();
     };
 
+    _p.exportZipInstance = function(masterName, instanceName, precision) {
```

Comment:
I'll think about this. The user interface in the browser, however, would preferably provide download of the data, so it would need to obtain the data object as it, not a path to it. A NodeJS command-line script would probably write the data to disk. The lowest common denominator seems to me to return the data, not to write it to a path.

The way you came from the perspective of the command-line script, I understand how this evolved. All methods in `MetapolatorProject` must be considered with great care, as this is the natural place to do thing that have no other home yet.
Take `exportInstance` it's not use for anything but CLI, it's maybe a bad role model. But if we add to it a `io` parameter, it may become a nice tool on that we can base an export to a zip file.